tox.ini

```diff
@@ -2,5 +2,9 @@
 envlist = py{36,37,38,39}
 
 [testenv]
-commands = py.test {posargs}
-deps = pytest
+deps = 
+    pytest
+    coverage
+commands = 
+    coverage erase
+    coverage run --source imaplib2 -m pytest {posargs}
```

Comment:
use pytest-cov (with pytest --cov) otherwise coverage run sets the sys.path incorrectly

Reply:
Also use a pyproject.toml with coverage config in:
Copy from here but with `source_pkgs=["imaplib2"]`
https://github.com/PyCQA/modernize/blob/master/pyproject.toml#L59-L70

Reply:
Done.